ns1/rest/zones.py

```diff
@@ -34,19 +40,40 @@ def import_file(
         self, zone, zoneFile, callback=None, errback=None, **kwargs
     ):
         files = [("zonefile", (zoneFile, open(zoneFile, "rb"), "text/plain"))]
+        params = self._buildImportParams(kwargs)
         return self._make_request(
             "PUT",
-            "import/zonefile/%s" % (zone),
+            "import/zonefile/{}".format(zone),
             files=files,
+            params=params,
             callback=callback,
             errback=errback,
         )
 
-    def create(self, zone, callback=None, errback=None, **kwargs):
+    # Extra import args are specified as query parameters not fields in a JSON object.
+    def _buildImportParams(self, fields):
+        params = {}
+        # Arrays of values should be passed as multiple instances of the same
+        # parameter but the zonefile API expects parameters containing comma
+        # seperated values.
+        if fields.get("networks") is not None:
+            networks_strs = map(str, fields["networks"])
```

Comment:
I think this is more Pythonic:
```suggestion
            networks_strs = [str(network) for network in fields["networks"]]
```